postfix/values.yaml

```diff
@@ -806,6 +806,20 @@ postfix:
     # sasl_passwd: |-
     #   [smtp.sendgrid.net]:587 yourSendGridUsername:yourSendGridPassword
 
+metrics:
+  enabled: false
+  image:
+    repository: chatwork/kumina-postfix-exporter
+    tag: k-kinzal-fixes-panic-logfilepath-empty
```

Comment:
https://github.com/chatwork/dockerfiles/pull/315
Due to this issue, the default is to use the docker image of the forked repository.